R/install.R

```diff
@@ -191,6 +191,8 @@ install <- function(packages = NULL,
     return(invisible(list()))
   }
 
+  libpath <- renv_path_pretty(renv_libpaths_active())
+  writef(header(sprintf("Installing package(s) into %s", libpath)))
```

Comment:
```suggestion
  writef(header(sprintf("Installing packages into %s", libpath)))
```

Reply:
I think it's okay to drop `(s)` here, since this reads like the name of a "stage" of the install process; that is, the "Installing packages" stage.